vagrant/deploy.sh

```diff
@@ -7,9 +7,8 @@ PROJECT_DIR=$(cd "${SCRIPT_DIR}/.." && pwd)
 cd "$SCRIPT_DIR"
 
 # Load environment variables
-export TARGET_CONCOURSE=bootstrap
 # shellcheck disable=SC2091
-$("${PROJECT_DIR}/concourse/scripts/environment.sh")
+$("${PROJECT_DIR}/concourse/scripts/environment.sh" "$@")
```

Comment:
Actually it does not work because the comment with the warning. It seems that all the part after the comment is ignored.

we could do this:
```
diff --git a/concourse/scripts/environment.sh b/concourse/scripts/environment.sh
index fc2261b..153d5e0 100755
--- a/concourse/scripts/environment.sh
+++ b/concourse/scripts/environment.sh
@@ -28,10 +28,12 @@ if [ -z "${CONCOURSE_ATC_PASSWORD:-}" ]; then
 fi
 
 cat <<EOF
-export AWS_ACCOUNT=${AWS_ACCOUNT}
+export AWS_ACCOUNT=${AWS_ACCOUNT};
 export DEPLOY_ENV=${DEPLOY_ENV}
 export CONCOURSE_ATC_USER=${CONCOURSE_ATC_USER}
-# WARNING: This is the password for the Bootstrap Concourse Lite in paas-bootstrap
+EOF
+echo '# WARNING: This is the password for the Bootstrap Concourse Lite in paas-bootstrap' 1>&2
+cat <<EOF
 export CONCOURSE_ATC_PASSWORD=${CONCOURSE_ATC_PASSWORD}
 export CONCOURSE_URL=${CONCOURSE_URL}
 export FLY_CMD=${FLY_CMD}
``` 

Or otherwise, simply remove the comment.